test/k8s-integration/driver.go

```diff
@@ -77,19 +76,25 @@ func installDriver(platform, goPath, pkgDir, stagingImage, stagingVersion, deplo
 	deployEnv = append(deployEnv,
 		fmt.Sprintf("GOPATH=%s", goPath),
 		fmt.Sprintf("GCE_PD_DRIVER_VERSION=%s", deployOverlayName))
-	deployCmd.Env = append(os.Environ(), deployEnv...)
+	deployEnv = append(os.Environ(), deployEnv...)
+	deployCmd.Env = deployEnv
 	err := runCommand("Deploying driver", deployCmd)
 	if err != nil {
-		return fmt.Errorf("failed to deploy driver: %v", err)
+		return fmt.Errorf("failed to deploy driver: %w", err)
 	}
-	klog.Infof("Deploying driver")
-	// TODO (#139): wait for driver to be running
+
+	var waitCmd *exec.Cmd
+	waitScript := filepath.Join(pkgDir, "deploy", "kubernetes", "wait-for-driver.sh")
 	if platform == "windows" {
-		klog.Infof("Waiting 15 minutes for the driver to start on Windows")
-		time.Sleep(15 * time.Minute)
+		klog.Infof("will wait for windows deployment")
+		waitCmd = exec.Command(waitScript, "--windows")
```

Comment:
Just a little clarification. for windows case, since we typically have a cluster with both linux and windows, we have daemonSets for both linux and windows deployed. 

In most of the cases, windows pods take more time linux pods. So when waiting for windows pods, linux pods should be ready. Or we can add wait for both just in case.

Reply:
Good point, will do.